mpire/comms.py

```diff
@@ -132,6 +132,7 @@ def init_comms(self) -> None:
 
         # Exception related
         self._exception_queue = self.ctx.JoinableQueue()
+        self._exception_received = self.ctx.Event(), self.ctx.Event()
```

Comment:
Could be slightly clearer by either adding a comment specifying the different events or simply adding one more variable for the progress bar event.

Still easy to understand with a quick scroll or ctrl+f so fine as is, just at first glance it's not clear.